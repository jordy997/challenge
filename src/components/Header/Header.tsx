import React from 'react'
import { View, Text, StyleSheet } from 'react-native'

const Header = () => {
    return (
        <View style={styles.header}>
            <Text style={styles.headerText}>
                Bienvenido de vuelta!
            </Text>
            <Text style={styles.points}>
                TUS PUNTOS
            </Text>
        </View>
    )
}

const styles = StyleSheet.create({
    header: {
        width: 290,
        marginTop: 45,
        left: 20,
    },
    headerText: {
        fontStyle: 'normal',
        fontSize: 20,
        lineHeight: 27,
        fontWeight: '800'
    },
    points: {
        fontWeight: '800',
        fontSize: 14,
        lineHeight: 19,
        color: '#9B9898',
        marginTop: 15,
        alignSelf: 'flex-start'
    }
})

export default Header