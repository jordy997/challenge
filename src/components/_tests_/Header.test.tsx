import React from 'react'

import { shallow } from 'enzyme'
import Header from '../Header/Header'

let wrapper

describe('Header component', () => {

    beforeEach(() => {
        wrapper = shallow(
            <Header />
        )
    })

    describe('Rendering', () => {
        it('Header should render correctly', () => {
            expect(wrapper).toMatchSnapshot()
        })
    })
})