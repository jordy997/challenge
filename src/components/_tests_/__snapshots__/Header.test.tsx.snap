// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Rendering Header should render correctly 1`] = `ShallowWrapper {}`;
